### PR TITLE
Provide maxquant dataset handler

### PR DESCRIPTION
**Pull request by sven1103**
If the dataset is of type `MaxQuantRunResult`, the correct registry is now determined and returned.

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -10,6 +10,7 @@ This project adheres to `Semantic Versioning <https://semver.org/>`_.
 **Added**
 
 * Add MaxQuantResultRegistry for MaxQuant analysis result datasets (`#12 <https://github.com/qbicsoftware/java-openbis-dropboxes/pull/12>`_)
+* Route MaxQuant result datasets to the `MaxQuantResultRegistry`
```
Comment (KochTobi) [inline CHANGELOG.rst:13]:
```suggestion

* Route MaxQuant result datasets to the `MaxQuantResultRegistry` (`#13 <https://github.com/qbicsoftware/java-openbis-dropboxes/pull/13/files>`_)
```
We need an empty line between each bullet point in RST

File: src/main/groovy/life/qbic/registration/handler/RegistrationHandler.groovy [hunk 2]
```diff
@@ -19,6 +21,7 @@ class RegistrationHandler {
      * <p>Currently supported dataset types:</p>
      * <ul>
      *     <li>{@link NfCoreResultRegistry}</li>
+     *     <li>{@link MaxQuantResultRegistry}</li>
```
Comment (KochTobi) [inline src/main/groovy/life/qbic/registration/handler/RegistrationHandler.groovy:24]: very nice :+1:

**Review by KochTobi — CHANGES_REQUESTED** [review]
Nice work! Please correct the changelog entry and you are good to go

**Review by KochTobi — APPROVED** [review]

**Comment by sven1103** [comment]
[MaxQuant: Extend Dataset handler for max quant(java dropbox)](https://trello.com/c/HeKY6eHM/855-maxquant-extend-dataset-handler-for-max-quantjava-dropbox)

**Pull request merged**